src/docs/Components/Breadcrumb/code.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Basic implementation
+
+### Import
+
+Import **the settings** and **the breadcrumb** scss files.
+
+```scss
+@import 'settings-tools/_all-settings';
+
+@include import-font-families();
+
+@import 'components/_c.breadcrumb';
+```
+
+### Usage
+
+You can create breadcrumbs in the following way:
+
+- Use the `mc-breadcrumb` class on an HTML element `<nav>` as a wrapper
+- Apply the `mc-breadcrumb__container` class to the `<ul>` container of your breadcrumb thread
+- The `<li>` elements must have the class `mc-breadcrumb__item`
+- Finally each link must have the class `mc-breadcrumb__link`
+
+```html
+<nav class="mc-breadcrumb" aria-label="Breadcrumb">
+  <ul class="mc-breadcrumb__container">
+    <li class="mc-breadcrumb__item">
+      <a href="#" class="mc-breadcrumb__link">Level 00</a>
+    </li>
+    <li class="mc-breadcrumb__item">
+      <span class="mc-breadcrumb__link" aria-current="page">Level 01</span>
+    </li>
+  </ul>
+</nav>
+```
+
+<Preview path="default" />
+
+## Variations
+
+### Color themes
+
+The breadcrumb is available in two colour variations.
+
+You can use these variations by applying the following modifiers:
+
+- **Light mode**: this is the default variation so you don't need to add a modifier
+- **Dark mode**: use the `mc-breadcrumb--dark` modifier
+
+#### Dark mode
+
+<Preview path="dark-mode" />
+
+## Behaviours
+
+### Smaller breakpoints
+
+As explained on [the main page](/Components/Breadcrumb/#smaller-breakpoints), the breadcrumb cannot be displayed entirely on lower resolutions.
+
+That's why you must make sure to display **only** the current page item on these resolutions.
+
+```html
+<nav class="mc-breadcrumb" aria-label="Breadcrumb">
+  <ul class="mc-breadcrumb__container">
+    <li class="mc-breadcrumb__item">
+      <a href="#" class="mc-breadcrumb__link">Level</a>
+    </li>
+  </ul>
+</nav>
+```
+
+If you don't have the possibility to perform tests allowing you to display only the current element, we provide you with a way to achieve the desired functioning in pure HTML & CSS:
+
+- Add the `mc-breadcrumb--responsive` modifier to the wrapper
+- Add the `is-active` class to the current page item.
+
+```html
+<nav class="mc-breadcrumb mc-breadcrumb--responsive" aria-label="Breadcrumb">
+  <ul class="mc-breadcrumb__container">
+    <li class="mc-breadcrumb__item">
+      <a href="#" class="mc-breadcrumb__link">Level 00</a>
+    </li>
+    <li class="mc-breadcrumb__item">
+      <a href="#" class="mc-breadcrumb__link">Level 01</a>
+    </li>
+    <li class="mc-breadcrumb__item is-active">
+      <span class="mc-breadcrumb__link" aria-current="page">Level 02</span>
+    </li>
+  </ul>
+</nav>
+```
+
+<Preview path="responsive" />
```

Comment:
Can we force the display of a `s-large` (for example) viewport size here?

Reply:
Unfortunately, at the moment, we cannot force a preview to be displayed the first time it is loaded. But I'm writing it down to see how we can set this up for the next few times.